proposals/extensions.md

```diff
@@ -144,7 +161,7 @@ The underlying type must include all the type parameters from the implicit exten
 
 ### Terminology
 
-We'll use "augments" for relationship to underlying type 
+We'll use "extends" for relationship to underlying/extended type 
```

Comment:
Is underlying and extended type the same thing? Should we rename "underlying type" to "extended type" (seems we are in process of doing that in the implementation)?

Reply:
Yes, they are the same and I'll align that in the spec first. For the implementation, I'm trying to be strategic to avoid noisy churn. Thanks